nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/stream.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ROOT=$(cd $(dirname $0)/..; pwd -P)
+CLASSPATH=$ROOT/lib/jcl-over-slf4j-1.7.21.jar:$ROOT/lib/slf4j-api-1.7.21.jar:$ROOT/lib/slf4j-log4j12-1.7.21.jar:$ROOT/lib/log4j-1.2.17.jar:commons-cli-1.4.jar:$ROOT/nxserver/lib/chronicle-queue-4.6.44.jar:$ROOT/nxserver/lib/zkclient-0.10.jar:$ROOT/nxserver/lib/kafka-clients-1.0.0.jar:$ROOT/nxserver/lib/kafka_2.11-1.0.0.jar:$ROOT/nxserver/lib/scala-library-2.11.7.jar:$ROOT/nxserver/lib/zookeeper-3.4.8.jar:$ROOT/nxserver/lib/nuxeo-stream-10.1-SNAPSHOT.jar
```

Comment:
`commons-cli-1.4.jar` is missing the `$ROOT/nxserver/lib/`.
You should write one per line for readability:

    CLASSPATH=\
    $ROOT/lib/jcl-over-slf4j-1.7.21.jar:\
    $ROOT/lib/slf4j-api-1.7.21.jar:\
    ...